src/converter/one/jfr/event/MethodTrace.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.jfr.event;
+
+public class MethodTrace extends Event {
```

Comment:
The class lacks overridden implementation for `sameGroup`, `hashCode` (needed for correct aggregation) and for `value` (required to support `--total` option).

Reply:
I did not say anything about `equals` - events are never checked for equivalence.

`hashCode` should not include `time` and `tid` - look how it's used in `EventAggregator`. I admit the name is misleading: it is actually a `groupHashCode` rather than instance `hashCode`.

I initially thought of including `method` in hashCode/sameGroup, but then realized that `method` is already a part of the event stack trace in our implementation. So, the default hashCode/sameGroup is sufficient. Sorry for confusion. Only `value` was missing, which you've now fixed.

Please make fields `public final` like in other events.

Reply:
On the other hand, `jfrconv` can be used with JDK Flight Recordings, where `MethodTrace` events do not include `method` in the stacktrace. So, the correct implementation of hashCode/sameGroup should take `method` into account. Check the implementation of `ContendedLock` - it's the closest to what you need here.

Reply:
Yeah, taking a fresh look at the code I see how confusing it is.
`EventAggregator.hashCode` + `EventAggregator.sameGroup` have a contract similar to the classic equals/hashCode.

`EventAggregator.hashCode/sameGroup` have parts common for all events plus they call `Event.hashCode/sameGroup` to compute event-specific part. But there is assymetry that causes confusion: `EventAggregator.sameGroup` checks `stackTraceId` in the common part, but when it comes to `hashCode`, `stackTraceId` is counted in event-specific implementation. This should be probably fixed, but not in this PR.